frontend/src/components/GifWidget/GifWidget.tsx

```diff
@@ -0,0 +1,84 @@
+import { SetStateAction, useEffect, useState } from "react";
+import {
+    GifWidgetStyled,
+    GifWidgetContent,
+    GifWidgetSearch,
+    GifList,
+    GifListImage,
+    GifWidgetAtrtibutionSection,
+    GiphyAttributionLogo
+} from "./GifWidget.styled";
+import axios from "axios";
+import GiphyAttributionLogoImage from "assets/GiphyAttributionLogo.png";
+
+const GifWidget = () => {
+    const [gifsLoading, setGifsLoading] = useState(true);
+    const [gifList, setGifList] = useState<any[]>([]);
```

Comment:
Remove unnecessary generic argument.

Reply:
Remove it, and create real type for that, "any" isn't good solution for that